CHANGELOG.md

```diff
@@ -6,6 +6,28 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Notable changes
+
+- `@cumulus/sync-granule` task should now properly handle
+syncing files from HTTP/HTTPS providers where basic auth is
+required and involves a redirect to a different host (e.g.
+downloading files protected by Earthdata login)
```

Comment:
```suggestion
downloading files protected by Earthdata Login)
```

Reply:
Branding 😄

Reply:
I'm not sure if we want to call it out here but there's a migration required. Maybe that's just highlighted in the release notes?

Reply:
What is the required migration?

Reply:
8bb83f8e44757bd4fc11e273fdc1092e394b94d7

@npauzenga I'm not quite sure what the required migration is though

Reply:
I guess it doesn't really matter. I meant the database migration: https://github.com/nasa/cumulus/pull/2356/files#diff-fa9a90bd6d6fb2969591fbf613f3e59e091d0e3b79afa889aea77ac7f6523aee

But those are just run on bootstrap without user intervention? So maybe we don't need to explicitly call these out when they come up?

Reply:
Yup, the data-persistence module automatically runs the db-migration module for the users, so they don't have to run it manually

https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence/postgres_migration.tf